flowexperimental/comp/wells/CompWell.hpp

```diff
@@ -66,7 +66,7 @@ class CompWell  : public CompWellInterface<TypeTag>
         static constexpr int num_phases = 2;
         std::array<EvalWell, num_phases> surface_densities_{0.};
         std::array<EvalWell, num_phases> volume_fractions_{0.};
-        std::array<std::array<EvalWell, num_comp>, num_phases> mass_fractions_{0.};
+        std::array<std::array<EvalWell, num_comp>, num_phases> mass_fractions_{};
```

Comment:
Is there a reason not to apply that transformation to the `surface_densities_` and `volume_fractions_` too?  Especially if the scalar type is something other than `double`?

Reply:
no reason other than them not causing warnings. amended.